src/dnddiceroller/Die.java

```diff
@@ -1,12 +1,51 @@
 package dnddiceroller;
 
+import java.util.Random;
+
 /**
- * A Die is a many sided object that when rolled provides a random value from
- * 1 through the number of sides on the object. Some dice are 6 sided and have
- * the numbers 1-6 on them. Some dice are 20 sided with the numbers 1-20 on them.
- * Others are called fudge dice and have the values of -1 0 or +1
- * @author Paul Scarrone
+ *
+ * @author Rhonda Marshall
+ * Oct 19, 2015
  */
 public class Die {
-  
-}
+    public int numberOfSides;             // number of sides on the dice
+    public int value;                     // value generated by a random number
```

Comment:
Only `final` fields should ever be `public` and in most cases they still should be `private`. We do not want to allow external modification of these after the class instance has been initialized otherwise the class could lose its context without us being able to track that modification.